From 8096444ac86e1ab06740d7f4d72e1e35eb963bb0 Mon Sep 17 00:00:00 2001
From: AsavarTzeth <asavartzeth@gmail.com>
Date: Thu, 15 Feb 2018 19:10:08 +0100
Subject: [PATCH 1/5] Rename indicator icons to match id

Ensure the uniqueness of the indicator icons, by prefixing the
filename with the application id.

This is required in order for indicator/tray icons to work with flatpak.
---
 COPYING                                       |  12 ++---
 share/CMakeLists.txt                          |  42 +++++++++---------
 ... => org.keeepassxc.KeePassXC.unlocked.png} | Bin
 ...k.png => org.keepassxc.KeePassXC.dark.png} | Bin
 ...png => org.keepassxc.KeePassXC.locked.png} | Bin
 ...k.png => org.keepassxc.KeePassXC.dark.png} | Bin
 ...png => org.keepassxc.KeePassXC.locked.png} | Bin
 ...g => org.keepassxc.KeePassXC.unlocked.png} | Bin
 ...k.png => org.keepassxc.KeePassXC.dark.png} | Bin
 ...png => org.keepassxc.KeePassXC.locked.png} | Bin
 ...g => org.keepassxc.KeePassXC.unlocked.png} | Bin
 ...k.png => org.keepassxc.KeePassXC.dark.png} | Bin
 ...png => org.keepassxc.KeePassXC.locked.png} | Bin
 ...g => org.keepassxc.KeePassXC.unlocked.png} | Bin
 ...k.png => org.keepassxc.KeePassXC.dark.png} | Bin
 ...png => org.keepassxc.KeePassXC.locked.png} | Bin
 ...g => org.keepassxc.KeePassXC.unlocked.png} | Bin
 ...k.png => org.keepassxc.KeePassXC.dark.png} | Bin
 ...png => org.keepassxc.KeePassXC.locked.png} | Bin
 ...g => org.keepassxc.KeePassXC.unlocked.png} | Bin
 ...k.png => org.keepassxc.KeePassXC.dark.png} | Bin
 ...png => org.keepassxc.KeePassXC.locked.png} | Bin
 ...g => org.keepassxc.KeePassXC.unlocked.png} | Bin
 ...svgz => org.keepassxc.KeePassXC.dark.svgz} | Bin
 ...gz => org.keepassxc.KeePassXC.locked.svgz} | Bin
 ... => org.keepassxc.KeePassXC.unlocked.svgz} | Bin
 src/core/FilePath.cpp                         |  12 ++---
 27 files changed, 33 insertions(+), 33 deletions(-)
 rename share/icons/application/128x128/apps/{keepassxc-unlocked.png => org.keeepassxc.KeePassXC.unlocked.png} (100%)
 rename share/icons/application/128x128/apps/{keepassxc-dark.png => org.keepassxc.KeePassXC.dark.png} (100%)
 rename share/icons/application/128x128/apps/{keepassxc-locked.png => org.keepassxc.KeePassXC.locked.png} (100%)
 rename share/icons/application/16x16/apps/{keepassxc-dark.png => org.keepassxc.KeePassXC.dark.png} (100%)
 rename share/icons/application/16x16/apps/{keepassxc-locked.png => org.keepassxc.KeePassXC.locked.png} (100%)
 rename share/icons/application/16x16/apps/{keepassxc-unlocked.png => org.keepassxc.KeePassXC.unlocked.png} (100%)
 rename share/icons/application/24x24/apps/{keepassxc-dark.png => org.keepassxc.KeePassXC.dark.png} (100%)
 rename share/icons/application/24x24/apps/{keepassxc-locked.png => org.keepassxc.KeePassXC.locked.png} (100%)
 rename share/icons/application/24x24/apps/{keepassxc-unlocked.png => org.keepassxc.KeePassXC.unlocked.png} (100%)
 rename share/icons/application/256x256/apps/{keepassxc-dark.png => org.keepassxc.KeePassXC.dark.png} (100%)
 rename share/icons/application/256x256/apps/{keepassxc-locked.png => org.keepassxc.KeePassXC.locked.png} (100%)
 rename share/icons/application/256x256/apps/{keepassxc-unlocked.png => org.keepassxc.KeePassXC.unlocked.png} (100%)
 rename share/icons/application/32x32/apps/{keepassxc-dark.png => org.keepassxc.KeePassXC.dark.png} (100%)
 rename share/icons/application/32x32/apps/{keepassxc-locked.png => org.keepassxc.KeePassXC.locked.png} (100%)
 rename share/icons/application/32x32/apps/{keepassxc-unlocked.png => org.keepassxc.KeePassXC.unlocked.png} (100%)
 rename share/icons/application/48x48/apps/{keepassxc-dark.png => org.keepassxc.KeePassXC.dark.png} (100%)
 rename share/icons/application/48x48/apps/{keepassxc-locked.png => org.keepassxc.KeePassXC.locked.png} (100%)
 rename share/icons/application/48x48/apps/{keepassxc-unlocked.png => org.keepassxc.KeePassXC.unlocked.png} (100%)
 rename share/icons/application/64x64/apps/{keepassxc-dark.png => org.keepassxc.KeePassXC.dark.png} (100%)
 rename share/icons/application/64x64/apps/{keepassxc-locked.png => org.keepassxc.KeePassXC.locked.png} (100%)
 rename share/icons/application/64x64/apps/{keepassxc-unlocked.png => org.keepassxc.KeePassXC.unlocked.png} (100%)
 rename share/icons/application/scalable/apps/{keepassxc-dark.svgz => org.keepassxc.KeePassXC.dark.svgz} (100%)
 rename share/icons/application/scalable/apps/{keepassxc-locked.svgz => org.keepassxc.KeePassXC.locked.svgz} (100%)
 rename share/icons/application/scalable/apps/{keepassxc-unlocked.svgz => org.keepassxc.KeePassXC.unlocked.svgz} (100%)

diff --git a/COPYING b/COPYING
index 19565055..6507e6ef 100644
--- a/COPYING
+++ b/COPYING
@@ -57,12 +57,12 @@ License: MIT
 
 Files: share/icons/application/*/apps/keepassxc.png
        share/icons/application/scalable/apps/keepassxc.svgz
-       share/icons/application/*/apps/keepassxc-dark.png
-       share/icons/application/scalable/apps/keepassxc-dark.svgz
-       share/icons/application/*/apps/keepassxc-locked.png
-       share/icons/application/scalable/apps/keepassxc-locked.svgz
-       share/icons/application/*/apps/keepassxc-unlocked.png
-       share/icons/application/scalable/apps/keepassxc-unlocked.svgz
+       share/icons/application/*/apps/org.keepassxc.KeePassXC.dark.png.png
+       share/icons/application/scalable/apps/org.keepassxc.KeePassXC.dark.png.svgz
+       share/icons/application/*/apps/org.keepassxc.KeePassXC.locked.png.png
+       share/icons/application/scalable/apps/org.keepassxc.KeePassXC.locked.png.svgz
+       share/icons/application/*/apps/org.keepassxc.KeePassXC.unlocked.png.png
+       share/icons/application/scalable/apps/org.keepassxc.KeePassXC.unlocked.png.svgz
        share/icons/application/*/mimetypes/application-x-keepassxc.png
        share/icons/application/scalable/mimetypes/application-x-keepassxc.svgz
 Copyright: 2016, Lorenzo Stella <lorenzo.stl@gmail.com>
diff --git a/share/CMakeLists.txt b/share/CMakeLists.txt
index 81bb2693..56c41419 100644
--- a/share/CMakeLists.txt
+++ b/share/CMakeLists.txt
@@ -61,51 +61,51 @@ add_custom_target(icons
 
                   # SVGZ to PNGs for KeePassXC
                   COMMAND inkscape -z -w 16 -h 16
-                    icons/application/scalable/apps/keepassxc-dark.svgz -e icons/application/16x16/apps/keepassxc-dark.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.dark.svgz -e icons/application/16x16/apps/org.keepassxc.KeePassXC.dark.png
                   COMMAND inkscape -z -w  24 -h 24
-                    icons/application/scalable/apps/keepassxc-dark.svgz -e icons/application/24x24/apps/keepassxc-dark.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.dark.svgz -e icons/application/24x24/apps/org.keepassxc.KeePassXC.dark.png
                   COMMAND inkscape -z -w  32 -h 32
-                    icons/application/scalable/apps/keepassxc-dark.svgz -e icons/application/32x32/apps/keepassxc-dark.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.dark.svgz -e icons/application/32x32/apps/org.keepassxc.KeePassXC.dark.png
                   COMMAND inkscape -z -w 48 -h 48
-                    icons/application/scalable/apps/keepassxc-dark.svgz -e icons/application/48x48/apps/keepassxc-dark.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.dark.svgz -e icons/application/48x48/apps/org.keepassxc.KeePassXC.dark.png
                   COMMAND inkscape -z -w 64 -h 64
-                    icons/application/scalable/apps/keepassxc-dark.svgz -e icons/application/64x64/apps/keepassxc-dark.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.dark.svgz -e icons/application/64x64/apps/org.keepassxc.KeePassXC.dark.png
                   COMMAND inkscape -z -w 128 -h 128
-                    icons/application/scalable/apps/keepassxc-dark.svgz -e icons/application/128x128/apps/keepassxc-dark.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.dark.svgz -e icons/application/128x128/apps/org.keepassxc.KeePassXC.dark.png
                   COMMAND inkscape -z -w 256 -h 256
-                    icons/application/scalable/apps/keepassxc-dark.svgz -e icons/application/256x256/apps/keepassxc-dark.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.dark.svgz -e icons/application/256x256/apps/org.keepassxc.KeePassXC.dark.png
 
                   # SVGZ to PNGs for KeePassXC
                   COMMAND inkscape -z -w 16 -h 16
-                    icons/application/scalable/apps/keepassxc-locked.svgz -e icons/application/16x16/apps/keepassxc-locked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.locked.svgz -e icons/application/16x16/apps/org.keepassxc.KeePassXC.locked.png
                   COMMAND inkscape -z -w  24 -h 24
-                    icons/application/scalable/apps/keepassxc-locked.svgz -e icons/application/24x24/apps/keepassxc-locked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.locked.svgz -e icons/application/24x24/apps/org.keepassxc.KeePassXC.locked.png
                   COMMAND inkscape -z -w  32 -h 32
-                    icons/application/scalable/apps/keepassxc-locked.svgz -e icons/application/32x32/apps/keepassxc-locked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.locked.svgz -e icons/application/32x32/apps/org.keepassxc.KeePassXC.locked.png
                   COMMAND inkscape -z -w 48 -h 48
-                    icons/application/scalable/apps/keepassxc-locked.svgz -e icons/application/48x48/apps/keepassxc-locked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.locked.svgz -e icons/application/48x48/apps/org.keepassxc.KeePassXC.locked.png
                   COMMAND inkscape -z -w 64 -h 64
-                    icons/application/scalable/apps/keepassxc-locked.svgz -e icons/application/64x64/apps/keepassxc-locked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.locked.svgz -e icons/application/64x64/apps/org.keepassxc.KeePassXC.locked.png
                   COMMAND inkscape -z -w 128 -h 128
-                    icons/application/scalable/apps/keepassxc-locked.svgz -e icons/application/128x128/apps/keepassxc-locked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.locked.svgz -e icons/application/128x128/apps/org.keepassxc.KeePassXC.locked.png
                   COMMAND inkscape -z -w 256 -h 256
-                    icons/application/scalable/apps/keepassxc-locked.svgz -e icons/application/256x256/apps/keepassxc-locked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.locked.svgz -e icons/application/256x256/apps/org.keepassxc.KeePassXC.locked.png
 
                   # SVGZ to PNGs for KeePassXC
                   COMMAND inkscape -z -w 16 -h 16
-                    icons/application/scalable/apps/keepassxc-unlocked.svgz -e icons/application/16x16/apps/keepassxc-unlocked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.unlocked.svgz -e icons/application/16x16/apps/org.keepassxc.KeePassXC.unlocked.png
                   COMMAND inkscape -z -w  24 -h 24
-                    icons/application/scalable/apps/keepassxc-unlocked.svgz -e icons/application/24x24/apps/keepassxc-unlocked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.unlocked.svgz -e icons/application/24x24/apps/org.keepassxc.KeePassXC.unlocked.png
                   COMMAND inkscape -z -w  32 -h 32
-                    icons/application/scalable/apps/keepassxc-unlocked.svgz -e icons/application/32x32/apps/keepassxc-unlocked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.unlocked.svgz -e icons/application/32x32/apps/org.keepassxc.KeePassXC.unlocked.png
                   COMMAND inkscape -z -w 48 -h 48
-                    icons/application/scalable/apps/keepassxc-unlocked.svgz -e icons/application/48x48/apps/keepassxc-unlocked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.unlocked.svgz -e icons/application/48x48/apps/org.keepassxc.KeePassXC.unlocked.png
                   COMMAND inkscape -z -w 64 -h 64
-                    icons/application/scalable/apps/keepassxc-unlocked.svgz -e icons/application/64x64/apps/keepassxc-unlocked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.unlocked.svgz -e icons/application/64x64/apps/org.keepassxc.KeePassXC.unlocked.png
                   COMMAND inkscape -z -w 128 -h 128
-                    icons/application/scalable/apps/keepassxc-unlocked.svgz -e icons/application/128x128/apps/keepassxc-unlocked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.unlocked.svgz -e icons/application/128x128/apps/org.keepassxc.KeePassXC.unlocked.png
                   COMMAND inkscape -z -w 256 -h 256
-                    icons/application/scalable/apps/keepassxc-unlocked.svgz -e icons/application/256x256/apps/keepassxc-unlocked.png
+                    icons/application/scalable/apps/org.keepassxc.KeePassXC.unlocked.svgz -e icons/application/256x256/apps/org.keepassxc.KeePassXC.unlocked.png
 
                   # SVGZ to PNGs for KeePassXC MIME-Type
                   COMMAND inkscape -z -w 16 -h 16
diff --git a/share/icons/application/128x128/apps/keepassxc-unlocked.png b/share/icons/application/128x128/apps/org.keeepassxc.KeePassXC.unlocked.png
similarity index 100%
rename from share/icons/application/128x128/apps/keepassxc-unlocked.png
rename to share/icons/application/128x128/apps/org.keeepassxc.KeePassXC.unlocked.png
diff --git a/share/icons/application/128x128/apps/keepassxc-dark.png b/share/icons/application/128x128/apps/org.keepassxc.KeePassXC.dark.png
similarity index 100%
rename from share/icons/application/128x128/apps/keepassxc-dark.png
rename to share/icons/application/128x128/apps/org.keepassxc.KeePassXC.dark.png
diff --git a/share/icons/application/128x128/apps/keepassxc-locked.png b/share/icons/application/128x128/apps/org.keepassxc.KeePassXC.locked.png
similarity index 100%
rename from share/icons/application/128x128/apps/keepassxc-locked.png
rename to share/icons/application/128x128/apps/org.keepassxc.KeePassXC.locked.png
diff --git a/share/icons/application/16x16/apps/keepassxc-dark.png b/share/icons/application/16x16/apps/org.keepassxc.KeePassXC.dark.png
similarity index 100%
rename from share/icons/application/16x16/apps/keepassxc-dark.png
rename to share/icons/application/16x16/apps/org.keepassxc.KeePassXC.dark.png
diff --git a/share/icons/application/16x16/apps/keepassxc-locked.png b/share/icons/application/16x16/apps/org.keepassxc.KeePassXC.locked.png
similarity index 100%
rename from share/icons/application/16x16/apps/keepassxc-locked.png
rename to share/icons/application/16x16/apps/org.keepassxc.KeePassXC.locked.png
diff --git a/share/icons/application/16x16/apps/keepassxc-unlocked.png b/share/icons/application/16x16/apps/org.keepassxc.KeePassXC.unlocked.png
similarity index 100%
rename from share/icons/application/16x16/apps/keepassxc-unlocked.png
rename to share/icons/application/16x16/apps/org.keepassxc.KeePassXC.unlocked.png
diff --git a/share/icons/application/24x24/apps/keepassxc-dark.png b/share/icons/application/24x24/apps/org.keepassxc.KeePassXC.dark.png
similarity index 100%
rename from share/icons/application/24x24/apps/keepassxc-dark.png
rename to share/icons/application/24x24/apps/org.keepassxc.KeePassXC.dark.png
diff --git a/share/icons/application/24x24/apps/keepassxc-locked.png b/share/icons/application/24x24/apps/org.keepassxc.KeePassXC.locked.png
similarity index 100%
rename from share/icons/application/24x24/apps/keepassxc-locked.png
rename to share/icons/application/24x24/apps/org.keepassxc.KeePassXC.locked.png
diff --git a/share/icons/application/24x24/apps/keepassxc-unlocked.png b/share/icons/application/24x24/apps/org.keepassxc.KeePassXC.unlocked.png
similarity index 100%
rename from share/icons/application/24x24/apps/keepassxc-unlocked.png
rename to share/icons/application/24x24/apps/org.keepassxc.KeePassXC.unlocked.png
diff --git a/share/icons/application/256x256/apps/keepassxc-dark.png b/share/icons/application/256x256/apps/org.keepassxc.KeePassXC.dark.png
similarity index 100%
rename from share/icons/application/256x256/apps/keepassxc-dark.png
rename to share/icons/application/256x256/apps/org.keepassxc.KeePassXC.dark.png
diff --git a/share/icons/application/256x256/apps/keepassxc-locked.png b/share/icons/application/256x256/apps/org.keepassxc.KeePassXC.locked.png
similarity index 100%
rename from share/icons/application/256x256/apps/keepassxc-locked.png
rename to share/icons/application/256x256/apps/org.keepassxc.KeePassXC.locked.png
diff --git a/share/icons/application/256x256/apps/keepassxc-unlocked.png b/share/icons/application/256x256/apps/org.keepassxc.KeePassXC.unlocked.png
similarity index 100%
rename from share/icons/application/256x256/apps/keepassxc-unlocked.png
rename to share/icons/application/256x256/apps/org.keepassxc.KeePassXC.unlocked.png
diff --git a/share/icons/application/32x32/apps/keepassxc-dark.png b/share/icons/application/32x32/apps/org.keepassxc.KeePassXC.dark.png
similarity index 100%
rename from share/icons/application/32x32/apps/keepassxc-dark.png
rename to share/icons/application/32x32/apps/org.keepassxc.KeePassXC.dark.png
diff --git a/share/icons/application/32x32/apps/keepassxc-locked.png b/share/icons/application/32x32/apps/org.keepassxc.KeePassXC.locked.png
similarity index 100%
rename from share/icons/application/32x32/apps/keepassxc-locked.png
rename to share/icons/application/32x32/apps/org.keepassxc.KeePassXC.locked.png
diff --git a/share/icons/application/32x32/apps/keepassxc-unlocked.png b/share/icons/application/32x32/apps/org.keepassxc.KeePassXC.unlocked.png
similarity index 100%
rename from share/icons/application/32x32/apps/keepassxc-unlocked.png
rename to share/icons/application/32x32/apps/org.keepassxc.KeePassXC.unlocked.png
diff --git a/share/icons/application/48x48/apps/keepassxc-dark.png b/share/icons/application/48x48/apps/org.keepassxc.KeePassXC.dark.png
similarity index 100%
rename from share/icons/application/48x48/apps/keepassxc-dark.png
rename to share/icons/application/48x48/apps/org.keepassxc.KeePassXC.dark.png
diff --git a/share/icons/application/48x48/apps/keepassxc-locked.png b/share/icons/application/48x48/apps/org.keepassxc.KeePassXC.locked.png
similarity index 100%
rename from share/icons/application/48x48/apps/keepassxc-locked.png
rename to share/icons/application/48x48/apps/org.keepassxc.KeePassXC.locked.png
diff --git a/share/icons/application/48x48/apps/keepassxc-unlocked.png b/share/icons/application/48x48/apps/org.keepassxc.KeePassXC.unlocked.png
similarity index 100%
rename from share/icons/application/48x48/apps/keepassxc-unlocked.png
rename to share/icons/application/48x48/apps/org.keepassxc.KeePassXC.unlocked.png
diff --git a/share/icons/application/64x64/apps/keepassxc-dark.png b/share/icons/application/64x64/apps/org.keepassxc.KeePassXC.dark.png
similarity index 100%
rename from share/icons/application/64x64/apps/keepassxc-dark.png
rename to share/icons/application/64x64/apps/org.keepassxc.KeePassXC.dark.png
diff --git a/share/icons/application/64x64/apps/keepassxc-locked.png b/share/icons/application/64x64/apps/org.keepassxc.KeePassXC.locked.png
similarity index 100%
rename from share/icons/application/64x64/apps/keepassxc-locked.png
rename to share/icons/application/64x64/apps/org.keepassxc.KeePassXC.locked.png
diff --git a/share/icons/application/64x64/apps/keepassxc-unlocked.png b/share/icons/application/64x64/apps/org.keepassxc.KeePassXC.unlocked.png
similarity index 100%
rename from share/icons/application/64x64/apps/keepassxc-unlocked.png
rename to share/icons/application/64x64/apps/org.keepassxc.KeePassXC.unlocked.png
diff --git a/share/icons/application/scalable/apps/keepassxc-dark.svgz b/share/icons/application/scalable/apps/org.keepassxc.KeePassXC.dark.svgz
similarity index 100%
rename from share/icons/application/scalable/apps/keepassxc-dark.svgz
rename to share/icons/application/scalable/apps/org.keepassxc.KeePassXC.dark.svgz
diff --git a/share/icons/application/scalable/apps/keepassxc-locked.svgz b/share/icons/application/scalable/apps/org.keepassxc.KeePassXC.locked.svgz
similarity index 100%
rename from share/icons/application/scalable/apps/keepassxc-locked.svgz
rename to share/icons/application/scalable/apps/org.keepassxc.KeePassXC.locked.svgz
diff --git a/share/icons/application/scalable/apps/keepassxc-unlocked.svgz b/share/icons/application/scalable/apps/org.keepassxc.KeePassXC.unlocked.svgz
similarity index 100%
rename from share/icons/application/scalable/apps/keepassxc-unlocked.svgz
rename to share/icons/application/scalable/apps/org.keepassxc.KeePassXC.unlocked.svgz
diff --git a/src/core/FilePath.cpp b/src/core/FilePath.cpp
index c6f1907a..d7f8a812 100644
--- a/src/core/FilePath.cpp
+++ b/src/core/FilePath.cpp
@@ -110,18 +110,18 @@ QIcon FilePath::trayIcon()
     bool darkIcon = useDarkIcon();
 
 #ifdef KEEPASSXC_DIST_SNAP
-    return (darkIcon) ? icon("apps", "keepassxc-dark", false) : icon("apps", "keepassxc", false);
+    return (darkIcon) ? icon("apps", "org.keepassxc.KeePassXC.dark", false) : icon("apps", "org.keepassxc.KeePassXC", false);
 #else
-    return (darkIcon) ? icon("apps", "keepassxc-dark") : icon("apps", "keepassxc");
+    return (darkIcon) ? icon("apps", "org.keepassxc.KeePassXC.dark") : icon("apps", "org.keepassxc.KeePassXC");
 #endif
 }
 
 QIcon FilePath::trayIconLocked()
 {
 #ifdef KEEPASSXC_DIST_SNAP
-    return icon("apps", "keepassxc-locked", false);
+    return icon("apps", "org.keepassxc.KeePassXC.locked", false);
 #else
-    return icon("apps", "keepassxc-locked");
+    return icon("apps", "org.keepassxc.KeePassXC.locked");
 #endif
 }
 
@@ -130,9 +130,9 @@ QIcon FilePath::trayIconUnlocked()
     bool darkIcon = useDarkIcon();
 
 #ifdef KEEPASSXC_DIST_SNAP
-    return darkIcon ? icon("apps", "keepassxc-dark", false) : icon("apps", "keepassxc-unlocked", false);
+    return darkIcon ? icon("apps", "org.keepassxc.KeePassXC.dark", false) : icon("apps", "org.keepassxc.KeePassXC.unlocked", false);
 #else
-    return darkIcon ? icon("apps", "keepassxc-dark") : icon("apps", "keepassxc-unlocked");
+    return darkIcon ? icon("apps", "org.keepassxc.KeePassXC.dark") : icon("apps", "org.keepassxc.KeePassXC.unlocked");
 #endif
 }
 
-- 
2.20.1

